python/runtime/cudaq/platform/py_alt_launch_kernel.cpp

```diff
@@ -493,6 +493,18 @@ py::object pyAltLaunchKernelR(const std::string &name, MlirModule module,
   return returnValue;
 }
 
+/// @brief Helper function to get boolean environment variable
+static bool getEnvBool(const char *envName, bool defaultVal = false) {
+  if (auto envVal = std::getenv(envName)) {
+    std::string tmp(envVal);
+    std::transform(tmp.begin(), tmp.end(), tmp.begin(),
+                   [](unsigned char c) { return std::tolower(c); });
+    if (tmp == "1" || tmp == "on" || tmp == "true" || tmp == "yes")
+      return true;
```

Comment:
Should this be `!defaultVal`? If the `defaultVal` is `true`, then there will be no distinction between success and failure.

Reply:
Hmm I presume env value, if defined, should take precedence over the default value? Updated.